applications/armored-chat/index.scss

```diff
@@ -104,8 +108,33 @@ body {
               width: auto;
               height: 100%;
 
+              // max-width: 400px;
+              // max-height: 300px;
+            }
+          }
+        }
+
+        .embeds {
+          width: 100%;
+
+          overflow-x: hidden;
+          a {
+            color: white;
+          }
+
+          .image-container {
+            width: 100%;
+            max-width: 400px;
+            max-height: 300px;
+
+            img {
+              // width: auto;
+              // height: 100%;
               max-width: 400px;
               max-height: 300px;
+
+              // max-width: 400px;
+              // max-height: 300px;
```

Comment:
And here.

Also, are px appropriate? What about 4k screens?

Reply:
The stylesheet is capping the preview image size and not the full size. You can see the full size images when you click on the embed link directly.